concepticondata/concepticon.tsv

```diff
@@ -2078,19 +2078,19 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2077	DILIGENT	Emotions and values	Being busy-working.	Property	
 2078	TEN THOUSAND	Quantity	The natural number ten thousand (10000).	Number	
 2079	CUNNING PERSON	Emotions and values	A smart and witty person.	Person/Thing	
-2080	BOWL (CLASSIFIER)	Quantity	classifier for bowl-like objects	Classifier	
-2081	CHICKEN (CLASSIFIER)	Quantity	classifier for bird-like animals	Classifier	
-2082	KNIFE (CLASSIFIER)	Quantity	classifier for knife-like objects	Classifier	
-2083	AXE (CLASSIFIER)	Quantity	classifier for axe-like objects	Classifier	
-2084	ROPE (CLASSIFIER)	Quantity	classifier for rope-like objects	Classifier	
-2085	STICK (CLASSIFIER)	Quantity	classifier for stick-like objects	Classifier	
-2086	GRAIN (CLASSIFIER)	Quantity	classifier for tiny objects, like rice	Classifier	
-2087	DROP (CLASSIFIER)	Quantity	classifier for small units of liquid, like water	Classifier	
-2088	CLOTHING (CLASSIFIER)	Quantity	classifier for things that one wears	Classifier	
-2089	PAIR (CLASSIFIER)	Quantity	classifier for objects that occur in pairs	Classifier	
+2080	BOWL (CLASSIFIER)	Quantity	Classifier for bowl-like objects	Classifier	
+2081	CHICKEN (CLASSIFIER)	Quantity	Classifier for bird-like animals	Classifier	
+2082	KNIFE (CLASSIFIER)	Quantity	Classifier for knife-like objects	Classifier	
+2083	AXE (CLASSIFIER)	Quantity	Classifier for axe-like objects	Classifier	
+2084	ROPE (CLASSIFIER)	Quantity	Classifier for rope-like objects	Classifier	
+2085	STICK (CLASSIFIER)	Quantity	Classifier for stick-like objects	Classifier	
+2086	GRAIN (CLASSIFIER)	Quantity	Classifier for tiny objects, like rice	Classifier	
+2087	DROP (CLASSIFIER)	Quantity	Classifier for small units of liquid, like water	Classifier	
+2088	CLOTHING (CLASSIFIER)	Quantity	Classifier for things that one wears	Classifier	
+2089	PAIR (CLASSIFIER)	Quantity	Classifier for objects that occur in pairs	Classifier	
 2090	MY	Possession	Belonging to me.	Other	
-2091	YOUR	Possession	belonging to the second person that is addressed by the first person	Other	732
-2092	HIS OR HER	Possession	belonging to the third person that is addressed	Other	
+2091	YOUR	Possession	Belonging to the second person that is addressed by the first person	Other	732
+2092	HIS OR HER	Possession	Belonging to the third person that is addressed	Other	
```

Comment:
Shouldn't these two and the classifiers also end in a dot?

Reply:
I found some others where "dots" are missing and added them.